coda/parser.py

```diff
@@ -90,7 +90,7 @@ def parse(self, value):
         """
         if not self.is_valid_coda(value):
             raise ValueError('The given value is not a valid coda content')
-        recordlist = unicode(value, 'windows-1252', 'strict').split('\n')
+        recordlist = value.decode('windows-1252', 'strict').split('\n')
```

Comment:
@sbidoul 
I don't really like to change the param type, but it can works in 2 and 3 if we ask for byte object.

In python2, it will change nothing, even if the type is str, so actual code in python 2 still works exactly the same:
`"test".decode('windows-1252', 'strict') == b"test".decode('windows-1252', 'strict')`

And in python3,  `b"test".decode('windows-1252', 'strict')` works too, so the param change is only applicable for python 3 code which isn't write yet.

Reply:
Didn't see work comment, sorry

Reply:
Yes, you don't change the param type. It's the caller that needs to pass bytes and needs to be adapted. What changed is that in python 2 open returned str/bytes, while in python 3 open returns unicode unless 'rb' is provided.